src/test/e2e/tests/specs.test.ts

```diff
@@ -135,14 +132,13 @@ describe('Finding a court or tribunal listing', () => {
     });
 
     it('should display 3 result', async() => {
-      expect(await hearingListPage.getResults()).toBe(3);
+      expect(await hearingListPage.getResults()).toBe(13);
     });
   });
 
   describe('Following the \'search\' path', () => {
-    const searchTerm = 'abergavenny';
-    const expectedNumOfResults = 1;
-    const expectedNumOfHearings = 1;
+    const searchTerm = 'Abergavenny Magistrates\' Court';
+    const expectedNumOfHearings = 13;
 
 
     it('should select \'tribunal hearing list\' option and navigate to search option page', async () => {
```

Comment:
update should to reflect hearings list page instead of search option page since that step has been removed